packages/http/src/router.ts

```diff
@@ -796,9 +796,6 @@ export class HttpRouter {
         }
 
         let matcher = `_path.startsWith(${JSON.stringify(prefix)}) && (_match = _path.match(${regexVar}))`;
-        if (!hasParameters) {
```

Comment:
This code was to improve performance. If there are really no parameter (`hasParameters` was obviously no good indicator for it), we do not need any expensive regex matching. With this PR, we have now for each route this expensive `_path.match` whether it has parameters or not. I think fixing `hasParameters` is a better idea.